dbschema/workflow-transition.esdl

```diff
@@ -0,0 +1,49 @@
+module ProgressReport {
+  type WorkflowTransition {
+    required to: ProgressReport::Status{
+      readonly := true;
+    };
+    required label: str {
+      readonly := true;
+    };
+    required type: WorkflowTransition::Type {
+      readonly := true;
+    };
+  }
+
+  type InternalTransition extending WorkflowTransition {
+    required name: WorkflowTransition::Name;
+    from: array<ProgressReport::Status> {
+      readonly := true;
+    };
+    `notify`: MembersWithRoles;
```

Comment:
@CarsonF - We're a little unsure if we've represented this correctly from the application code.  Thoughts?